bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -45,6 +45,26 @@
 		<description>Suppress all songs from the current queue</description>
 	</channel-type>
 
+	<channel-type id="codec" advanced="true">
+		<item-type>String</item-type>
+		<label>Codec</label>
+		<description>Name of codec currently being decoded</description>
+		<state readOnly="true">
+			<options>
```

Comment:
Please add the tag notifying that the state is not limited to the provided list of options. Sorry, I don't remember its exact name but probably something like `limitedToOptions` (to be checked)